src/Microsoft.SqlTools.Credentials/Program.cs

```diff
@@ -36,15 +36,12 @@ internal static void Main(string[] args)
                 string logFilePath = commandOptions.LogFilePath;
```

Comment:
This code is common to the 3 backend services: resourceprovider, credential service and tools service layer. We should probably refactor it out into a programbase class, so we do not leave behind unintended code in one when we fix others. Filed issue #712 for this.

Reply:
Since an issue has been filed we will deal with this in a future PR.